packages/editor/CodeMirror/editorCommands/swapLine.test.ts

```diff
@@ -0,0 +1,32 @@
+import { EditorView } from '@codemirror/view';
+import { EditorSelection } from '@codemirror/state';
+import swapLine from './swapLine';
+
+
+describe('swapLine', () => {
+	it('should swap line down', () => {
+		const initialText = 'Hello\nWorld\nJoplin\n';
+		const editorView = new EditorView({
+			doc: initialText,
+			selection: EditorSelection.cursor(0),
+		});
+
+		swapLine(1)(editorView);
```

Comment:
```suggestion
		swapLine(SwapLineDirection.Down)(editorView);
```

Please use the `enum` here.

Reply:
updated it!